### ISSUE TITLE
How to perform "Rotate"  with out showing controls in the UI?

### ISSUE THREAD

**Issue by arunakommuri** [Yalantis/uCrop#428]
**Do you want to request a *feature* or report a *bug*?**

**What is the current behavior?**

**What is the expected behavior?**

**If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem.**

**Please attach any *image files*, *URL* and `stack trace` that can be used to reproduce the *bug*.**

**Which versions of uCrop, and which Android API versions are affected by this issue? Did this work in previous versions of uCrop?**

**Comment by Legementarion** [comment]
@arunakommuri  Hello! Ucrop have option - setHideBottomControls, try this to hide bottom controls

**Comment by arunakommuri** [comment]
yeah, But i want to achieve rotate functionality with out showing the controls..

**Comment by Legementarion** [comment]
@arunakommuri So? Do you want hide only rotate control, but show all others?

**Comment by arunakommuri** [comment]
@Legementarion No, I want to hide all the controls not only rotate, But i need the rotate functionality. Is their any possibility to do..

**Comment by sleepstream** [comment]
I think that it'll be better to have possibility to hide only one button (one by one) and all together.
Now i hide panel and use 
`options.setAllowedGestures(UCropActivity.SCALE, UCropActivity.SCALE, UCropActivity.SCALE);`

But rotate doesn't work.

**Comment by krida2000** [comment]
I use `options.setAllowedGestures(UCropActivity.ALL, UCropActivity.ALL, UCropActivity.ALL)` and both rotating and scaling work.
